kodilla-good-paterns/src/main/java/com/kodilla/good/patterns/challenges/NewSale.java

```diff
@@ -0,0 +1,15 @@
+package com.kodilla.good.patterns.challenges;
+
+import java.time.LocalDate;
+
+public class NewSale implements SaleRepository {
+
+    @Override
+    public void createSaleRepository(User user, Product product, LocalDate localDate) {
```

Comment:
create sale repository? Tworzysz tu repozytorium? A jesli tworzysz, to czemu void?